protos/local/workflow_commands.proto

```diff
@@ -137,4 +138,16 @@ message ContinueAsNewWorkflowExecution {
 
 /// Indicate a workflow has completed as cancelled. Generally sent as a response to an activation
 /// containing a cancellation job.
-message CancelWorkflowExecution {}
\ No newline at end of file
+message CancelWorkflowExecution {}
+
+/// A request to set/check version information for a certain change.
+/// TODO: Add table of outcomes
+message HasChange {
```

Comment:
nit. I'm not sure about this name as it's a bit too passive, maybe `ApplyChange` would be better name for a job as it would assume that something needs to be done with it?